modules/git/git_commits/exercise_slides/contents.tex

```diff
@@ -0,0 +1,120 @@
+\title{%
+  Good commits - Exercises
+}
+\author{Amanda Berg}
+\institute{%
+  KTH EECS
+}
+
+\mode<article>{\maketitle}
+\mode<presentation>{%
+  \begin{frame}
+    \maketitle
+  \end{frame}
+}
+
+\mode*
+
+
+\section{Exercise 1 - Commit messages}
+\begin{frame}
+\begin{block}{Sum over list}
+    We wish to add up all non-negative numbers in a list. We have the following piece of code:
```

Comment:
They haven't learned any programming yet (very first week). But they have covered the terminal, so we can have shell scripts as examples.

Reply:
Hm, they should be able to handle it. You give them everything.

Reply:
Not sure how you want to do with commit plans in the second exercise, if you want to build on this. Then there might be problems.

Counting the number of files in the current working directory and its subdirectories as the example?
```bash
find . -type f | wc -l
```
The bug could be to forget the `-l` option to `wc`. Or miss to add the `-type f` to `find`.